src/main/java/creativei/manager/impl/InquiryManagerImpl.java

```diff
@@ -1,29 +1,57 @@
 package creativei.manager.impl;
 
 import creativei.entity.Inquiry;
+import creativei.entity.InquiryAddress;
+import creativei.helper.ResponseHelper;
 import creativei.manager.InquiryManager;
 import creativei.service.BranchService;
+import creativei.service.InquiryAddressService;
 import creativei.service.InquiryService;
+import creativei.service.impl.InquiryAddressServiceImpl;
+import creativei.vo.AddressVo;
+import creativei.vo.InquiryVo;
 import creativei.vo.ResponseObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
+import java.io.PrintWriter;
 import java.util.List;
 
 /**
  * Created by user on 12/16/2017.
  */
+@Service
 public class InquiryManagerImpl implements InquiryManager {
+   private static final Logger logger= LoggerFactory.getLogger(InquiryManagerImpl.class);
     @Autowired
     InquiryService inquiryService;
+    @Autowired
+    InquiryAddressService inquiryAddressService;
 
     @Override
     public ResponseObject getAll() {
         return null;
     }
 
     @Override
-    public ResponseObject create(Inquiry inquiry) {
-        return null;
+    public ResponseObject create(InquiryVo inquiryVo) {
+     try{
+
+            Inquiry inquiry=new Inquiry(inquiryVo);
+            inquiry= inquiryService.create(inquiry);
+            inquiryVo= ResponseHelper.getCreateInquiryResponseData(inquiry,inquiryVo);
+            InquiryAddress inquiryAddress=new InquiryAddress(inquiryVo.getAddress());
+            AddressVo addressVo=inquiryVo.getAddress();
```

Comment:
No need to create inquiryAddress separately. Also reponse should also be created in a single method.